DllImportGenerator/DllImportGenerator/Marshalling/Forwarder.cs

```diff
@@ -1,23 +1,104 @@
 ﻿using System;
 using System.Collections.Generic;
-
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
 
 namespace Microsoft.Interop
 {
-    internal class Forwarder : IMarshallingGenerator
+    internal class Forwarder : IMarshallingGenerator, IAttributedReturnTypeMarshallingGenerator
     {
         public TypeSyntax AsNativeType(TypePositionInfo info)
         {
             return info.ManagedType.AsTypeSyntax();
         }
 
+        private bool TryRehydrateMarshalAsAttribute(TypePositionInfo info, out AttributeSyntax marshalAsAttribute)
+        {
+            marshalAsAttribute = null!;
```

Comment:
Since we're compiling against netstandard2.0, we don't have the nullable analysis attributes to correctly represent the `Try`-pattern, so the easiest way to get a good experience is to do what I'm doing here.

Reply:
Ah, right, thanks. Forgot we don't have those attributes.